support-frontend/assets/pages/supporter-plus-landing/supporterPlusLanding.tsx

```diff
@@ -150,6 +163,24 @@ export function SupporterPlusLandingPage({
 	};
 	const heading = <LandingPageHeading />;
 
+	const participations = initAbTests(
+		detectCountry(),
+		countryGroupId,
+		getSettings(),
+	);
+	const [supportOnceDisplay, setSupportOnceDisplay] = useState(
+		participations.singleLessProminent === 'variant',
+	);
+
+	const dispatch = useContributionsDispatch();
+	function onSupportOnceContainerClick(displaySupportOnce: boolean) {
+		setSupportOnceDisplay(
+			participations.singleLessProminent !== 'variant' ||
+				(supportOnceDisplay && displaySupportOnce),
+		);
+		dispatch(setProductType('ONE_OFF'));
```

Comment:
I think this can be simplified. 

Firstly, I'm not sure it's possible for `onSupportOnceContainerClick` to be called where `displaySupportOnce` will be `true`? I can't see a scenario where a click event on this element would lead us to `setSupportOnceDisplay` to `false`. Therefore I don't think then we need this `displaySupportOnce` parameter.

I'm also not sure why we're checking `participations.singleLessProminent !== 'variant'` here? When will a user who is not in the `variant` of `singleLessProminent` be able to click on the `SupportOnceContainer`? It should only be visible to users in the  the `variant` of `singleLessProminent`?

I think we could just be calling `setSupportOnceDisplay(false)` instead of...

```
setSupportOnceDisplay(
  participations.singleLessProminent !== 'variant' ||
    (supportOnceDisplay && displaySupportOnce),
);
```

Reply:
Ahhh, if you get the SupportOnceContainerClick event you must have the SupportOnce component open therefore no need at all to check for the variant. Once depressed once simply set to false why pass arguments through. Ok changes made.